Hazel/src/Hazel/Utility/Random.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+#include <random>
+
+namespace Hazel {
+
+	class Random
+	{
+	public:
+		/*
+		* Returns a Random Real between min and max
+		*/
+		template<typename T>
+		static T Real(T min, T max)
+		{
+			HZ_CORE_ASSERT(min < max, "min is > max");
```

Comment:
if `min < max` then `min == max` is also a possibility. Should be more like `"min is >= max"`

Reply:
Oh yep! i miss that one